src/components/layouts/EmptyState/DashboardEmptyState.tsx

```diff
@@ -0,0 +1,50 @@
+import { PrimaryBtn } from '@/components/buttons/PrimaryBtn'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { AddIcon, TasksListIcon } from '@/icons'
+import { setShowModal } from '@/redux/features/createTaskSlice'
+import store from '@/redux/store'
+import { UserType } from '@/types/interfaces'
+import { SxCenter } from '@/utils/mui'
+import { Box, Stack, Typography } from '@mui/material'
+
+const DashboardEmptyState = ({ userType }: { userType: UserType }) => {
+  return (
+    <AppMargin size={SizeofAppMargin.LARGE} py="20px">
+      <Box
+        sx={{
+          display: 'flex',
+          height: '80vh',
+          ...SxCenter,
+        }}
+      >
+        <Stack rowGap={'20px'} direction={'column'} sx={{ width: '453px' }}>
+          <Stack rowGap={'12px'} direction={'column'}>
+            <TasksListIcon />
+
+            <Typography variant="lg" lineHeight={'32px'}>
+              {userType == UserType.INTERNAL_USER ? " You don't have any tasks yet" : 'No tasks assigned'}
```

Comment:
Font size of this header is `heading2Xl` in the Figma design. Here we are giving `lg` i.e. 16px only.